Packs/Zabbix/pack_metadata.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "Zabbix",
+    "description": "Allow integration with Zabbix api.",
+    "support": "xsoar",
+    "serverMinVersion": "5.0.0",
```

Comment:
`serverMinVersion` and `fromVersion` in integration should be aligned. Either 4.5.0 all of them or 5.0.0